src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/CpxVariantDetector.java

```diff
@@ -879,6 +852,153 @@ private static boolean alignmentIsDisjointFromAlphaOmega(final SimpleInterval al
 
     // =================================================================================================================
 
+    /**
+     * Two Cpx are equal if they are the same in
+     * <ul>
+     *     <li>
+     *         affected range
+     *     </li>
+     *     <li>
+     *         alt haplotype sequence
+     *     </li>
+     *     <li>
+     *         segments
+     *     </li>
+     *     <li>
+     *         alt arrangements
+     *     </li>
+     * </ul>
+     * @return
+     */
+    @DefaultSerializer(EquivKey.Serializer.class)
+    private static final class EquivKey {
```

Comment:
Can you give this class a name that is more descriptive of what it represents? I'm thinking something like `CpxVariantCanonicalRepresentation` or something like that. Make sense to you?